dimod/sampleset.py

```diff
@@ -1187,6 +1187,89 @@ def truncate(self, n, sorted_by='energy'):
         return type(self)(record, self.variables, copy.deepcopy(self.info),
                           self.vartype)
 
+    def slice(self, *slice_args, **kwargs):
+        """Create a new SampleSet with rows sliced according to standard `slice`
+        syntax.
+
+        Args:
+            start (int, optional, default=None):
+                Start index for `slice`.
+
+            stop (int):
+                Stop index for `slice`.
+
+            step (int, optional, default=None):
+                Step value for `slice`.
+
+            sorted_by (str/None, optional, default='energy'):
+                Selects the record field used to sort the samples before
+                truncating. Note that sample order is maintained in the
```

Comment:
Users who don't know the structure with an underlying array will be confused over whether the original or sorted order is the order that will be maintained going forward. "The original sample order is maintained after slicing." seems clearer to me.

Reply:
You're right, thanks, I fixed this.

Reply:
re: `slice`, I want to refer users to the [`slice`](https://docs.python.org/3/library/functions.html#slice) built-in.